traitquestdev/companylogin.php

```diff
@@ -21,19 +32,24 @@
 </head>
 
 <body>
-	<h1>Admin Login</h1>
-	<form id="formCompanyLogin" class="form" method="post">
-		<div id="columnCompany">
-			<input type="text" name="company" id="companyName" class="inputForm" placeholder="Company Name" />
-		</div>
-		<div id="columnEmail">
-			<input type="text" name="email" id="email" class="inputForm" placeholder="Email" />
-		</div>
-		<div id="columnPassword">
-			<input type="password" name="password" id="password" class="inputForm" placeholder="Password" />
-		</div>
-		<div id="loginResponse"></div>
-		<input type="submit" name="submit" id="loginSubmit" class="buttonForm" value="Login" />
-	</form>
+<div id="wrapper">
+	<?php include "header.php"; ?>
+	<div id="mainContainer" class="container">
+		<h1 class="text-center">Admin Login</h1>
```

Comment:
no need title, the Submit button is clear enough to tell that it's login for admin